open_discussions/views.py

```diff
@@ -29,7 +29,6 @@ def index(request, **kwargs):  # pylint: disable=unused-argument
     user = None
     auth = request.COOKIES.get(api_settings.JWT_AUTH_COOKIE)
     site_key = settings.OPEN_DISCUSSIONS_DEFAULT_SITE_KEY
-
     if request.user.is_authenticated:
```

Comment:
This is an unrelated change, could you remove it? I also personally much prefer this spacing being here because it makes the code more readable to me and I had just added it back in a few commits ago.